symplyphysics/laws/electricity/current_through_element_is_proporional_to_voltage_und_reversly_proportional_to_resistance.py

```diff
@@ -0,0 +1,21 @@
+from symplyphysics import (
+    symbols, Eq, pretty, solve, Quantity, units,
+    validate_input, validate_output, expr_to_quantity
+)
+
+# Description
+## Ohm's law - I = U / R
+## Current flowing through the resistor is proportional to applied voltage and reversly proportional to impedance of that resistor
```

Comment:
Follow the description template of other laws. Description comes first. All law parameters have description.